uber/site_sections/preregistration.py

```diff
@@ -522,7 +522,7 @@ def pay_for_extra_members(self, session, payment_id, stripeToken):
     def transfer_badge(self, session, message='', **params):
         old = session.attendee(params['id'])
 
-        assert old.is_transferable, 'This badge is not transferrable'
+        assert old.is_transferable, 'This badge is not transferrable at this time. Badge transfers are temporarily disabled for hotel early booking. We hope to re-enable this feature as soon as we can. Thank you for your patience.'
```

Comment:
Oh no, we don't want this message to say this anymore! I think you'll have to just manually change it back.